ssl/ssl_init.c

```diff
@@ -195,11 +194,14 @@ int OPENSSL_init_ssl(uint64_t opts, const OPENSSL_INIT_SETTINGS * settings)
         return 0;
     }
 
-    if (!RUN_ONCE(&ssl_base, ossl_init_ssl_base))
+    if (!OPENSSL_init_crypto(opts
+                             | OPENSSL_INIT_LOAD_CONFIG
```

Comment:
This is at least  not consistent with the documentation as far as I understand. So maybe the documentation should get updated?

Reply:
Done. I updated the pod documentation and also added a CHANGES entry.